content/en/blog/_posts/2019-12-21-Kubernetes-on-MIPS.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: blog
+title: "Kubernetes on MIPS"
+date: 2019-12-21
+slug: Kubernetes-on-MIPS
+---
+
+**Authors:** TimYin Shi, Dominic Yin, Wang Zhan, Jessica Jiang, Will Cai, Jeffrey Gao, Simon Sun (Inspur)
+
+## Background
+
+[MIPS](https://en.wikipedia.org/wiki/MIPS_architecture) (Microprocessor without Interlocked Pipelined Stages) is a reduced instruction set computer (RISC) instruction set architecture (ISA), appeared in 1981 and developed by MIPS Technologies. Now MIPS architecture is widely used in many electronic products.
+
+[Kubernetes](https://kubernetes.io) has officially supported a variety of CPU architectures such as x86, arm/arm64, ppc64le, s390x. However it's a pity that Kubernetes doesn't support MIPS. With the widespread use of cloud native technology, users under MIPS architecture also have an urgent demand for Kubernetes on MIPS.
+
+## Achievements
+
+For many years, in order to enrich the ecology of the open source community, we have been working on adapting Kubernetes to the MIPS architecture. With the continuous iterative optimization and the performance improvement of MIPS CPU, we have made some breakthrough progresses on mips64el platform.
+
+Over the years, we have been actively participated in the Kubernetes community and have rich experience in the using and optimization of Kubernetes technology. Recently, we tried to adapt Kubernetes to the MIPS architecture platform and gained a stage achievement. Successfully completed the migration and adaptation of Kubernetes and related components, not only built a stable and highly available MIPS cluster, but also completed the conformance test of kubernetes-v1.16.2.
+
+![Kubernetes on MIPS](/images/blog/2019-12-21-Kubernetes-on-MIPS/kubernetes-on-mips.png)
+
+*Figure 1 Kubernetes on MIPS*
+
+## K8S-MIPS component build
+
+Almost all native cloud components related to Kubernetes do not provide MIPS version installation package or image. The prerequisite of deploying Kubernetes on MIPS platform is to compile and build all required components on mips64el platform. These components include:
```

Comment:
``suggestion
Almost all native cloud components related to Kubernetes do not provide a MIPS version installation package or image. The prerequisite of deploying Kubernetes on the MIPS platform is to compile and build all required components on the mips64el platform. These components include:
```